### ISSUE TITLE
Spec should define a processing model

### ISSUE THREAD

**Issue by clelland** [WICG/deprecation-reporting#8]
From https://github.com/w3c/reporting/issues/169#issuecomment-568517978:

> We should define some processing algorithm for each of the above? We describe what the reports are, but there is no guidance on processing.

This spec describes a report format, but no processing model for generating or sending them. That should be described in some amount of useful detail.

cc: @igrigorik

**Comment by brentfulgham** [comment]
Indeed. WebKit would be interested in adopting this API, but it's hard to do so without a good plan for when reports should be generated.
